Git 23 (#3510)
platformsh/platformsh-docs: sites/platform/src/define-routes/_index.md

and any additional routes (such as redirecting HTTP to HTTPS) are created.
This means the final generated routes differ by environment and so shouldn't be hard coded in your app.
These routes are available in the `PLATFORM_ROUTES` environment variable as a base64-encoded JSON object.

> They are also available in your `/run/config.json` file so you can maintain a high number of routes more easily.

Reviewer: ```suggestion
They are also available within an application container from the `/run/config.json` file so you can maintain a high number of routes more easily.
```
